### ISSUE TITLE
Continuation of #427: ktlint and ruleset logging

### ISSUE THREAD

**Issue by akuleshov7** [pinterest/ktlint#818]
Guys, small question related to #427. We are now seeing code for logging like this in ktlint: 
```
if (debug) {
    println("[DEBUG] ")
}
```
It is enabled by --debug option. 

Also in some rules we see following code:
```
        // run `KTLINT_DEBUG=experimental/indent ktlint ...` to enable debug output
            private val debugMode =
            (System.getenv("KTLINT_DEBUG") ?: "").split(",").contains("experimental/indent")
```
So it is enabled by KTLINT_DEBUG sys property.

In our rule set (https://github.com/cqfn/diKTat) we are trying to use sl4fj.

**So questions are:**
1) Is it ok if we will contribute and merge these two ways of enabling debug mode?
2) Is it ok  if we will contribute and add slf4j logging in your project?

**Comment by Tapchicoma** [comment]
Yes, there are related ticket - https://github.com/pinterest/ktlint/issues/589

Regarding logging framework - my idea is to use https://github.com/MicroUtils/kotlin-logging

**Comment by paul-dingemans** [comment]
Fixed by #1327 in version 0.44.0
